frontend/src/classes/SpatialChat.ts

```diff
@@ -48,5 +48,5 @@ export class ChatEntry {
 
 export enum ChatEditorType {
     DEFAULT_EDITOR = 1,
-    RICH_TEXT_EDITOR
+    RICH_TEXT_EDITOR = 0
```

Comment:
Could you elaborate on why this change was required? (Just out of curiosity). [Enums](https://www.typescriptlang.org/docs/handbook/enums.html) in typescript auto-increment. So `RICH_TEXT_EDITOR` in was previously 2. Also, the code should require these values to be hard-coded anywhere ideally.